src/wallet/wallet2.cpp

```diff
@@ -5710,6 +5727,12 @@ void wallet2::get_payments_out(std::list<std::pair<crypto::hash,wallet2::confirm
   }
 }
 //----------------------------------------------------------------------------------------------------
+void wallet2::get_payments_out(std::list<std::pair<crypto::hash,wallet2::confirmed_transfer_details>>& confirmed_payments,
+    uint64_t min_height, uint64_t max_height, const boost::optional<uint32_t>& subaddr_account, const std::set<uint32_t>& subaddr_indices) const
+{
+  get_payments_out_min_height_inclusive(confirmed_payments, m_confirmed_txs, min_height, max_height, subaddr_account, subaddr_indices);
+}
```

Comment:
Since you're breaking compatibility anyway, please remove the "min_height_inclusive" suffix.

Reply:
Done (0c43135599c1a3cbf1dd9d50085ab03e5a46812d).